composer.json

```
@@ -24,7 +24,7 @@
         "egulias/email-validator": "^2.0",
         "erusev/parsedown": "^1.7",
         "league/flysystem": "^1.0.8",
-        "monolog/monolog": "^1.12",
+        "monolog/monolog": "^1.12 || ^2.0",
```

Please use `^1.12|^2.0` notation for constraints.

```suggestion
        "monolog/monolog": "^1.12|^2.0",
```

---

done